.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "blocks/socketio"]
```

Comment:
Haaaaaaaaands (accidentally added nio blocks for the project while in the wrong directory)